.storybook/pages/FeedbackOverview/FeedbackOverview.tsx

```diff
@@ -194,7 +194,7 @@ export const FeedbackOverview = ({ activeIndex = 0 }: Props) => (
                                 flexShrink: 0,
                               }}
                             >
-                              1
+                              &nbsp;
```

Comment:
Could we maybe use `map` to render all of these instead of specifying each one? Kinda annoying that you had to go through and replace every number in a million little boxes.

Reply:
regex find/replace to the rescue.  find `(?<=flexShrink: 0,\n\s*}}\n\s*>\n\s*)\d+(?=\n\s*<\/div>)` replace `&nbsp;` 🤣